legend-sdlc-test-generation/src/test/java/org/finos/legend/sdlc/test/junit/TestServiceTestGenerator.java

```diff
@@ -59,6 +59,29 @@ public void testTestService2()
                 "testTestSuites::TestService2");
     }
 
+
+    @Test
+    public void testTestLongServiceName()
```

Comment:
You also need tests for names with unicode characters that require multiple bytes to encode, including one that spans the byte limit.